packages/lesswrong/components/community/modules/CommunityMembersFullMap.tsx

```diff
@@ -0,0 +1,39 @@
+import { Components, registerComponent } from '../../../lib/vulcan-lib';
+import React from 'react';
+import { createStyles } from '@material-ui/core/styles';
+import { getSearchClient } from '../../../lib/algoliaUtil';
+import { Configure, InstantSearch } from 'react-instantsearch-dom';
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  map: {
+    height: '100vh',
+    marginTop: -50,
+    [theme.breakpoints.down('sm')]: {
+      marginTop: 0,
+    },
+  }
+}))
+
+
+const CommunityMembersFullMap = ({classes}: {
+  classes: ClassesType,
+}) => {
+  const { SearchResultsMap } = Components
+  
+  return <InstantSearch
+    indexName={'test_users'}
+    searchClient={getSearchClient()}
+  >
+    <SearchResultsMap zoom={1} className={classes.map} />
+    <Configure hitsPerPage={1000} filters="_geoloc.lat>-100" />
```

Comment:
I'm not sure what a "page" means in this context, but does this mean it will show only 1000 people? I think possibly we should do more – the intent of this page seems to be that it shows literally everyone.